### PR TITLE
Add different highlighting to clarify when data is not applicable

### LINKED ISSUE #318: Indicate lack of data/applicability for process nodes for certain filters
image
I'm concerned people are going to interpret this as "the selected country can't do this process" when really it means "we don't apply this kind of highlighting to processes as a rule, just inputs and stages." How difficult would it be to apply some sort of no-data indicator to the circled areas whenever a highlighting filter (ie every filter but specific input) is active? Like a background of gray diagonal lines or something

### LINKED ISSUE #363: add more padding under legend when legend is active
image

note also that the weird bolding continues for me...I dunno

### PR DESCRIPTION
**Pull request by niharikasingh**
Closes #318 
Closes #363

### CODE REVIEW

**Comment by jmelot** [comment]
Code looks fine to me, and I like the way the hatching looks/don't find it too busy!

I am unsure about the pipe as a separator. Could you please try either putting the "data not available" part of the legend on a new line, or separating it from the rest of the legend with some whitespace? Also, could you disallow linebreaks between the box and the text?

Screen Shot 2022-11-07 at 10 37 22 AM

@za158 right now the hatching appears for market concentration and supplier countries. Did we want to apply it to supplier companies as well?

**Comment by za158** [comment]
Yeah
---
Zachary Arnold
Analytic Lead, Emerging Technology Observatory
Center for Security and Emerging Technology
***@***.***

**Comment by niharikasingh** [comment]
Re: supplier companies top-level highlighting. We actually do highlight top-level nodes and stages for the company filter. Do we want to ignore this data and instead use the cross-hatching? 

Screen Shot 2022-11-07 at 11 37 23 PM

**Comment by jmelot** [comment]
I think it would be nice to have the cross-hatching for nodes where no company provision information is available, but keep the highlighting where it _is_ present, but defer to @za158

**Comment by za158** [comment]
Yeah, I was thinking about this last week actually, and agree with Jennifer. I'm pretty sure the issue is limited to the design nodes (only place where we have supplier information at the process level)

File: supply-chain/src/components/graph_node.js [hunk 1]
```diff
@@ -14,11 +14,22 @@ const GraphNode = (props) => {
...
+    }
+    return false;
+  }
+
   const getHighlightClass = () => {
     if (node in highlights && highlights[node] !== 0) {
       return " highlighted " + getBackgroundGradient(highlights[node], highlights);
     } else if ((highlighterFilter === FILTER_CONCENTRATION) || (highlighterFilter === FILTER_COUNTRY)) {
       return " not-applicable";
+    } else if (highlighterFilter === FILTER_ORG && !_isDesignProcessNode()) {
```
Comment (jmelot) [inline supply-chain/src/components/graph_node.js:29]: This is a bit fragile - we could add data for other nodes later (or remove it from these nodes). How much harder would it be to apply the styling based on whether data is present for the node or not?

**Review by jmelot — APPROVED** [review]
👍 !

**Pull request merged**